swingset-demo/src/main/java/com/nqadmin/swingset/demo/Example4UsingHelper.java

```diff
@@ -225,6 +222,14 @@ public JMenuBar getJMenuBar() {
 		// nothing to do...
 		return null;
 	}
+	
+	@Override
+	protected void populateSSComboBoxes() {
```

Comment:
Classes using SSFormViewScreenHelper should now implement populateSSComboBoxes() so that we can minimize the chance that something will be bound before setOptions() is called.